interface-elements-for-web/articles/grid/customization-dialog/filtering-page.md

```diff
@@ -4,6 +4,24 @@ author: Anna Kondratova
 legacyId: 118822
 ---
 # Filtering Page
-The "Filtering" page allows you to create and change the filter conditions. For each column, select the criteria operator and specify the operand value.
+The **Filtering** page allows you to create and change filter conditions. 
+
+For each column, you can select the criteria operator and specify the operand value.
```

Comment:
You can select the criteria operator and specify the operand value for each column.